pandas/core/strings.py

```diff
@@ -2172,9 +2172,9 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
 
         Returns
         -------
-        concat : str if `other is None`, Series/Index of objects if `others is
-            not None`. In the latter case, the result will remain categorical
-            if the calling Series/Index is categorical.
+        concat :
+            str if `others` is None, Series/Index of objects if `others` is not
+            None.
```

Comment:
1) I would still put all possible data-types right after the `concat: ` part.

2) OCD kicking in here: can we rephrase so that `None` isn't by itself?

Reply:
Fair enough. See new suggestion